src/client/pages/Forever/components/Intro/IntroStories.tsx

```diff
@@ -124,63 +124,61 @@ interface IntroProps extends RouteComponentProps {
   referrerName: string
 }
 
-export const IntroStoriesComponent: React.FC<IntroProps> = ({
+const incrementPage = (page: number, pages: JSX.Element[]) => {
```

Comment:
This name imo indicates side effects of the function while it's actually pure. Maybe we should name it something like `getNextPageNumber` or something?

Reply:
That's a very good point! Will change it!